docs/sql-reference/sql-functions/date-time-functions/makedate.md

```diff
@@ -0,0 +1,72 @@
+# makedate
+
+## Description
+
+Reduces the specified date and time by a specified number of hours.
+
+## Syntax
+
+```Haskell
+DATE makedate(INT year, INT dayOfYear);
+```
+
+## Parameters
+
+* `year`: Year ranges from 0 to 9999.The supported data type is INT. 
+
+* `dayOfYear`: The day of the year. The supported data type is INT.
+
+## Return value
+
+Returns a date, given year and day-of-year values.
+day-of-year must be greater than 0 or the result is NULL. 
```

Comment:
day-of-year must be between 1 and the number of days in the current year (365 in normal years, 366 in leap years), otherwise the result is NULL.